zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/message/ProcessMessageSubscriptionCorrelateProcessor.java

```diff
@@ -96,9 +96,11 @@ final var record = command.getValue();
 
     if (subscription == null) {
       rejectCommand(command, RejectionType.NOT_FOUND, NO_SUBSCRIPTION_FOUND_MESSAGE);
+      return;
 
     } else if (subscription.isClosing()) {
       rejectCommand(command, RejectionType.INVALID_STATE, ALREADY_CLOSING_MESSAGE);
+      return;
 
     } else if (hasAlreadyBeenCorrelated(record, subscription)) {
```

Comment:
🙃 The else-ifs can now just be regular ifs

Reply:
🤡 This is one fewer line per block:

```
if (..) {
  return;
} else if (..) {
///
```

compared to:

```
if (..) {
  return;
}
if (..) {
///
```

Reply:
You mean you don't get paid per line? 🫢